VoteStakeRegistry/actions/getGrantInstruction.ts

```diff
@@ -86,9 +84,10 @@ export const getGrantInstruction = async ({
         voterWeightRecord: voterWeightPk,
         vault: voterATAPk,
         depositToken: fromPk,
-        authority: grantAuthority,
-        payer: feePayerPk,
+        tokenAuthority: grantAuthority,
```

Comment:
This is governance right? `grantAuthority` is confusing name here because it can be any governance